Shapes.java

```diff
@@ -1,26 +1,34 @@
-import java.util.*;
-
-public class Shapes{
+import java.lang.Math;
+class AreaCalculator{
   public static void main(String[] args) {
-    Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
-    double side = input.nextDouble();
-    square(side);
-  }
+      squareAreaCalculator(6);
```

Comment:
I like all your variable and method names here! They clearly show what they do/what value they're storing.